test/testinput/bufr_gpsztd.yaml

```
@@ -0,0 +1,95 @@
+# (C) Copyright 2024 NOAA/NWS/NCEP/EMC
+# (C) Copyright 2024 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "./gdas.gpsipw.t00z.20241213.bufr"
+
+      exports:
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+
+          latitude:
+            query: "*/CLATH"
+
+          longitude:
+            query: "*/CLONH"
+
+          stationName:
+            query: "*/STSN"
+
+          heightOfStation:
+            query: "*/SELV"
+
+#         # ObsValue
+#         precipitableWater:
+#           query: "*/TPWT"
+#           type: float
+
+          # ObsValue
+          zenithTotalDelay:
+            query: "*/TDEL"
```

```suggestion
            query: "*/APDS"
```

I've tried both `TDEL` and `APDS` both yield:
```
PRE-MAIN-WARNING Warning: Query String */APDS didn't apply to subset NC012004
```

---

@BenjaminRuston 
I ran a print_queries.x on your bufr file, which showed me the pathways of the mnemonics. TDEL does not exist in bufr table and therefore it will not print anything. APDS, while it does exist, is not in the query path you gave it. The full query path for ADPS is `*/GNSSRPSQ/APDS` . Try changing your path to that in the yaml and running again. It appears this only happens once so I don't believe you need to use "group_by" either.

---

yes this is the correct line and produces output

this definitely **is** being used by NOAA GSL and discussions with them now am cooking ;0)

thanks @nicholasesposito